site/docs/guidelines/tables-lists.mdx

```diff
@@ -21,6 +21,14 @@ Tables and lists have distinct patterns which shouldn't be mixed.
 - Lists are sortable by a ‘Sorted by’ [Select](/components/select), and do not have column header labels
 - A single row of a list should make sense in isolation
 
+### [Focus indicator](/guidelines/interaction-states)
+- Lists: 
+    -   Single Action: Exactly one interactive element, on keyboard navigation show focus around the whole row. See also: [Single action Tile](/components/tile/#singleactiontile-not-built).
+    -   Multi-action: Multiple interactive elements inside the object container, on keyboard navigation show focus on the individual interactive elements. See also: [Multi-action Tile](/components/tile/#singleactiontile-not-built)
+- Table: On keyboard navigation show focus on the individual interactive elements within table cells.
```

Comment:
I feel like this information will flow better if we move it to the bottom of the page, after tables and lists have been explained.

Reply:
I split it up and put the list and table focus indicator guidelines in their respective sections.